ritwik12_Celestial-bodies-detection_app_views_46.py

```
@@ -0,0 +1,107 @@
+            buf = BytesIO()
+            filext = request.files[form.image_file.name].filename.split('.')[-1].upper()
+            if(filext=='JPG'):
+                filext='JPEG'
+            im_resize.save(buf, format=filext)
+            # get back bytes
+            imageBytes = buf.getvalue()
+            print("using form")
+
+        elif form.image_url.data:
+            # from url
+            response = requests.get(form.image_url.data)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)